get-data.R

```diff
@@ -18,34 +18,38 @@ library(janitor)
 library(stringr)
 
 
-## data frame of dip metadata records in the B.C. Data Catalogue
-dip_records_df <- bcdc_list_group_records("data-innovation-program") 
-
-
-## function to concatenate all metadata resources (files) for one record 
+## function to concatenate all metadata resources (files) for one bcdc record 
 concatenate_all_record_resources <- function(record){
   
+record_title <- bcdc_get_record(record) %>%
```

Comment:
I think you could speed this up significant if you call `bcdc_get_record` once and then plunk that resulting object into subsequent calls of both `bcdc_tidy_resources` and `bcdc_get_data`. This will eliminate multiple calls to the catalogue for the same information (i.e. the record).

Reply:
Oh excellent catch, tx @boshek.